modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/accounts.json

```
@@ -1,22 +1,102 @@
 [
 	{
 		"externalReferenceCode": "RAYACCOUNT-001",
-		"name": "Test Account Raylife 1",
+		"name": "Aloysius Ridsdale",
 		"type": "business"
 	},
 	{
 		"externalReferenceCode": "RAYACCOUNT-002",
-		"name": "Test Account Raylife 2",
+		"name": "Loydie Roskrug",
 		"type": "business"
 	},
 	{
 		"externalReferenceCode": "RAYACCOUNT-003",
-		"name": "Test Account Raylife 3",
+		"name": "Jolie Zecchi",
 		"type": "business"
 	},
 	{
 		"externalReferenceCode": "RAYACCOUNT-004",
-		"name": "Test Account Raylife 4",
+		"name": "Pincus Middup",
+		"type": "business"
+	},
+	{
+		"externalReferenceCode": "RAYACCOUNT-005",
+		"name": " Shurlocke Letessier",
```

Please, remove this space.
```suggestion
		"name": "Shurlocke Letessier",
```

---

OK